data/src/test/java/com/teamagam/gimelgimel/data/message/repository/cache/room/dao/SearchMessagesDaoTest.java

```diff
@@ -16,8 +28,7 @@
 import org.robolectric.RuntimeEnvironment;
 
 import static com.teamagam.gimelgimel.data.common.DbTestUtils.getDB;
-import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.mock;
+import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(RobolectricTestRunner.class)
 public class SearchMessagesDaoTest extends BaseTest {
```

Comment:
After you'll rebase this on develop (in the future, not now...), remember that this test-class should be refactored to use the newly introduced junit rule that initialize the DB for you.

Reply:
mabey a todo: shuld be used there then?